### PR TITLE
Protein translation

### PR DESCRIPTION
**Pull request by annaoliinyk**
https://exercism.org/tracks/python/exercises/protein-translation

### CODE REVIEW

File: protein_translation/protein_translation_task.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            "UCU": "Serine",
+            "UCC": "Serine",
+            "UCA": "Serine",
+            "UCG": "Serine",
+            "UAU": "Tyrosine",
+            "UAC": "Tyrosine",
+            "UGU": "Cysteine",
+            "UGC": "Cysteine",
+            "UGG": "Tryptophan"
+        }
```
Comment (AnaKovaGH) [inline protein_translation/protein_translation_task.py]: Put it into separate constant outside the function

File: protein_translation/protein_translation_task.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Task from https://exercism.org/tracks/python/exercises/protein-translation
+
+def proteins(strand):
+    codons_list = get_codors_list(strand)
+    proteins_list = []
+    for codon in codons_list:
+        if codon in ["UAA", "UAG", "UGA"]:
```
Comment (AnaKovaGH) [inline protein_translation/protein_translation_task.py]: Put this list in a constant outside the function

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was commented above

**Comment by annaoliinyk** [comment]
Added main(), logging and renamed constants

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**